src/access/workflows/AaveWithdrawWorkflow.sol

```diff
@@ -23,16 +23,21 @@ contract AaveWithdrawWorkflow {
     IPoolAddressesProvider public immutable poolAddressProvider;
     /// @notice Address of the Bridger contract
     IBridger public immutable bridger;
+    /// @notice Address of the Safe contract
+    address public immutable safe;
+    /// @notice Fee charged upon withdrawal. 10bps.
+    uint256 public constant FEE = 1e15;
 
     /* ============ Constructor ============ */
 
     /**
      * @notice Initializes the contract with Aave's pool address provider
      * @param poolAddressProvider_ The address of Aave's pool address provider
      */
-    constructor(address poolAddressProvider_, address bridger_) {
+    constructor(address poolAddressProvider_, address bridger_, address safe_) {
```

Comment:
to change the fee we will just upgrade?

Reply:
Yes. We can make it a param on a AccessManager.